sdk/scheduler/src/main/java/com/mesosphere/sdk/state/DefaultStateStore.java

```diff
@@ -1,417 +0,0 @@
-package com.mesosphere.sdk.state;
```

Comment:
Reviewer note: this was moved to `StateStore.java` without further changes (outside of a new TODO around task name retrieval in `storeStatus()`)